cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -147,7 +147,7 @@
                 this._setSubnav(draggable.find('> .cms-dragitem .cms-submenu'));
 
                 // adds event for hiding the subnav
-                draggable.bind('mouseenter mouseleave mouseover', function (e) {
+                draggable.bind('pointerenter pointerleave pointerover', function (e) {
```

Comment:
tried not to change much for the moment, only relevant parts. cleanup of this thing will follow, same way we did for modal